File: app/src/main/java/com/alvinhkh/buseta/view/MainActivity.java
package com.alvinhkh.buseta.view;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.alvinhkh.buseta.C;
import com.alvinhkh.buseta.ui.search.SearchActivity;

public class MainActivity extends AppCompatActivity {

    // legacy placeholder activity
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Intent intent = new Intent();
        intent.setClass(this, com.alvinhkh.buseta.ui.MainActivity.class);
        if (getIntent().getExtras() != null) {
            intent.putExtras(getIntent().getExtras());
        }
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/alvinhkh/buseta/view/MainActivity.kt
package com.alvinhkh.buseta.view

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    // legacy placeholder activity
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent()
        intent.setClass(this, com.alvinhkh.buseta.ui.MainActivity::class.java)
        if (getIntent().extras != null) {
            intent.putExtras(getIntent().extras!!)
        }
        startActivity(intent)
        finish()
    }
}
